srcpkgs/tageditor/template

```diff
@@ -0,0 +1,63 @@
+# Template file for 'tageditor'
+pkgname=tageditor
+version=3.3.4
+revision=1
+build_style=cmake
+configure_args="-DBUILD_SHARED_LIBS:BOOL=ON"
+makedepends="tagparser-devel c++utilities-devel"
+short_desc="Tag editor for AAC, ID3, Vorbis, Opus, FLAC and Matroska (CLI only)"
+maintainer="Andrew J. Hesford <ajh@sideband.org>"
+license="GPL-2.0-only"
+homepage="https://github.com/Martchus/${pkgname}"
```

Comment:
Don't use variables in homepage, is should be copyable from template.

Reply:
Done, here and in all other templates in the pull request